public/index.html

```diff
@@ -12,22 +12,33 @@
     <link href="/css/main.css" rel="stylesheet" />
 </head>
 <body>
-    <header class="navbar navbar-inverse navbar-fixed-top">
-        <div id="cross-site" class="nav-tabs">
+    <header class="navbar navbar-inverse navbar-static-top">
+        <div id="cross-site" class="nav-tabs navbar-collapse collapse">
             <div class="container" role="navigation">
                 <ul class="nav nav-tabs">
                     <li role="presentation"><a href="https://www.histwest.org.au/" class="cross-site-link">Website</a></li>
                     <li role="presentation"><a href="https://shop.histwest.org.au/" class="cross-site-link">Shop</a></li>
-                    <li role="presentation" class="active"><a href="https://research.histwest.org.au/" class="cross-site-link">Research</a></li>
+                    <li role="presentation" class="active"><a href="https://research.histwest.org.au/" class="cross-site-link">Collections</a></li>
                 </ul>
             </div>
         </div>
         <div class="container">
-            <a href="/"><img src="/images/logo-small.png" alt="Royal Western Australian Historical Society" /></a>
+            <a href="/"><img src="/images/rwahs-logo_web.png" alt="Royal Western Australian Historical Society" class="logo" /></a>
```

Comment:
You've added a new logo image under a different filename, but the old one is still there.  Either rename yours to `logo-small.png` (which will overwrite the old one), or just delete the old one.  Not a huge fan of using both hyphen and underscore in the same filename either, but meh.